File: TASK-03/ SubtasK 2/main.go
package main

import (
	"fmt"
	"io/ioutil"
	"os"
)

func main() {
	
	inputFile, err := os.Open("input.txt")
	if err != nil {
		fmt.Println("Error opening input file:", err)
		return
	}
	defer inputFile.Close()

	content, err := ioutil.ReadAll(inputFile)
	if err != nil {
		fmt.Println("Error reading from input file:", err)
		return
	}

	
	outputFile, err := os.Create("output.txt")
	if err != nil {
		fmt.Println("Error creating output file:", err)
		return
	}
	defer outputFile.Close()

	_, err = outputFile.Write(content)
	if err != nil {
		fmt.Println("Error writing to output file:", err)
		return
	}

	fmt.Println("File content copied successfully.")
}

File: TASK-03/ SubtasK 2/main.rs
use std::fs;
use std::io::{self, Write};

fn main() -> io::Result<()> {

    let content = fs::read_to_string("input.txt")?;

 
    fs::write("output.txt", content)?;

    println!("File content copied successfully.");
    Ok(())
}

